src/components/TextInput/BaseTextInput/index.native.tsx

```diff
@@ -21,36 +35,70 @@ import useTheme from '@styles/themes/useTheme';
 import useThemeStyles from '@styles/useThemeStyles';
 import variables from '@styles/variables';
 import CONST from '@src/CONST';
-import * as baseTextInputPropTypes from './baseTextInputPropTypes';
-
-function BaseTextInput(props) {
+import type BaseTextInputProps from './types';
+
+function BaseTextInput(
+    {
+        label = '',
+        // name = '',
+        value = undefined,
+        defaultValue = undefined,
+        placeholder = '',
+        errorText = '',
+        icon = null,
+        textInputContainerStyles = [],
+        containerStyles = [],
+        inputStyle = [],
+        forceActiveLabel = false,
+        autoFocus = false,
+        disableKeyboard = false,
+        autoGrow = false,
+        autoGrowHeight = false,
+        hideFocusedState = false,
+        maxLength = undefined,
+        hint = '',
+        // shouldSaveDraft = false,
+        onInputChange = () => {},
+        shouldDelayFocus = false,
+        submitOnEnter = false,
+        multiline = false,
+        // shouldUseDefaultValue = false,
+        shouldInterceptSwipe = false,
+        autoCorrect = true,
+        prefixCharacter,
+        inputID,
+        ...inputProps
+    }: BaseTextInputProps,
+    ref: ForwardedRef<HTMLFormElement | Component<AnimatedProps<TextInputProps>, unknown, unknown>>,
```

Comment:
Could you clarify why you need this `unknown` in typing?

Reply:
Actually its not needed

Reply:
@kubabutkiewicz Then why did you leave it? 😅